src/tests/ftest/harness/timeout.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from time import sleep
+
+from apricot import Test
+
+
+class HarnessTimeoutTest(Test):
+    """Advanced harness test cases.
+
+    :avocado: recursive
+    """
+
+    def tearDown(self):
+        """Tear down after each test case."""
+        super().tearDown()
+
+        # Sleep almost the full 60 seconds to verify that this method is not
+        # interrupted
+        self.log.info("Verifying that timeout is given 60 seconds to execute")
+        self.log.info("Sleeping for 59 seconds")
+        sleep(59)
+
+    def test_timeout(self):
```

Comment:
Does this duplicate https://github.com/daos-stack/daos/blob/1a4b9197ad277fb750c8b77662094582cb715bb9/src/tests/ftest/avocado_tests.py#L63-L69?